public_html/wp-content/themes/pattern-directory/src/components/pattern-thumbnail/index.js

```diff
@@ -35,9 +34,7 @@ function PatternThumbnail( { pattern, showAvatar, showOptions } ) {
 			<div className="pattern-grid__pattern-frame">
 				<a href={ pattern.link } rel="bookmark">
 					<span className="screen-reader-text">{ decodeEntities( pattern.title.rendered ) }</span>
-					<Disabled>
-						<Canvas url={ addQueryArgs( pattern.link, { view: true } ) } />
-					</Disabled>
+					<Canvas url={ addQueryArgs( pattern.link, { view: true, modified: pattern.modified_gmt } ) } />
```

Comment:
`modified` will break the cache when a pattern is modified, but there should probably also be a site-wide cachebuster in case we update styles in the pattern preview endpoint.

Reply:
I wouldn't be opposed to a manual cache break if it's really needed in short order.